python/docs/source/reference/pyspark.sql/catalog.rst

```diff
@@ -0,0 +1,48 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+=======
+Catalog
+=======
+
+.. currentmodule:: pyspark.sql
+
+.. autosummary::
+    :toctree: api/
+
+    Catalog.cacheTable
```

Comment:
Here it doesn't have `Catalog` class but others like DataFrame or SparkSession have them.

Reply:
ditto as https://github.com/apache/spark/pull/36647#discussion_r880061516.

Let me remove `DataFrame`, in `dataframe.rst` too.